pytripgui/model/plot_model.py

```diff
@@ -10,38 +10,82 @@
 
 class ProjectionSelector:
     def __init__(self):
-        self.current_x_slice = 50
-        self.current_y_slice = 125
-        self.current_z_slice = 125
+        self.current_T_slice = 0
+        self.current_S_slice = 0
+        self.current_C_slice = 0
+
+        self.max_T_depth = 0
+        self.max_S_depth = 0
+        self.max_C_depth = 0
 
         # "Transversal" (xy)
         # "Sagittal" (yz)
         # "Coronal"  (xz)
         self.plane = "Transversal"
 
     def next_slice(self):
+        if self.position < self._max_position():
+            self.position += 1
+        else:
+            self.position = 0
```

Comment:
Can you add some comment ? What does it mean that position is at zero ? Is it geometrical center of the patient volume ?

Reply:
Convention of slice numbering cannot be deduced from the code itself, hence this fragment is not self-documented and require some external comment explaining this issue.

Reply:
What about applying modulo operation and some meaningful comment ?
Do you mean by this code that it should cycle around numbers between 0 and (max_position - 1) ?

```
self.position = (self.position + 1) % self._max_position()
```